askama_parser/src/node.rs

```diff
@@ -53,27 +72,13 @@ impl<'a: 'l, 'l> Node<'a> {
     }
 
     fn many(i: &mut InputStream<'a, 'l>) -> ParseResult<'a, Vec<Box<Self>>> {
-        let mut nb_nodes = 0;
-
-        repeat(0.., move |i: &mut _| Self::parse_nodes(i, &mut nb_nodes))
-            .map(|v: Vec<_>| v)
```

Comment:
I don't remember why this "custom type ascription" was needed, but I'm happy that it isn't anymore!